mockup/patterns/relateditems/pattern.js

```diff
@@ -350,6 +353,7 @@ define([
       self.treeQuery = new utils.QueryHelper(
         $.extend(true, {}, self.options, {
           pattern: self,
+          vocabularyUrl: self.options.treeOptions.vocabularyUrl,
```

Comment:
Can we use the whole treeOptions object here instead of just taking vocabularyUrl?